src/test/java/bio/terra/common/fixtures/DaoOperations.java

```diff
@@ -105,7 +106,7 @@ public Snapshot ingestSnapshot(Snapshot snapshot) {
   public Snapshot createAndIngestSnapshot(Dataset dataset, String snapshotPath) throws IOException {
     SnapshotRequestModel snapshotRequest = createSnapshotRequestFromDataset(dataset, snapshotPath);
     Snapshot snapshot =
-        createSnapshotFromSnapshotRequest(snapshotRequest, dataset.getProjectResourceId());
+        createSnapshotFromSnapshotRequest(snapshotRequest, dataset, dataset.getProjectResourceId());
```

Comment:
probably doesn't really matter, but if we are passing in the dataset now, we could just get the project resource Id from the dataset in the method instead of passing it in as well

Reply:
Good point!